### ISSUE TITLE
Overlapping Work

### ISSUE THREAD

**Issue by kordianbruck** [MusicConnectionMachine/VisualizationG6#11]
Hey,

just as a suggestion: it does not make sense that both teams do a fully fleched Web App with all components. It would be better if one of the team focuses on the big picture, like the main framework and detailed views while the other team comes up with a great idea how to navigate the "social network graph". 

So: maybe you can coordinate a bit with your colleges so we don't end up with two systems that will run in parallel. Ideally we can maybe integrate one into the other.

@MusicConnectionMachine/group-5 https://github.com/MusicConnectionMachine/VisualizationG5/issues/14

**Comment by chaoran-chen** [comment]
Hey G6! How are you? Do you have some ideas how we can coordinate it? As I can see in your setup-branch, you are also going to use React - I think that's great! Do you want to build a SPA or just to use the components to build widgets?

Our current plan is to build a React-bundle for each component which can be included in our express pages as well as on external websites. Maybe you could use our existing Express setup and focus on the development of components?

@MusicConnectionMachine/group-5

**Comment by sacdallago** [comment]
Sounds like a very good plan! I suggest the two teams chat a bit more about this :) And then, obviously, update us!

**Comment by demirdagemir** [comment]
Hey @MusicConnectionMachine/group-5 ,

What you suggest sounds cool. We also discussed that we will create bundles for each component. I'm not really experienced with Express, let me(us) look into it, discuss it with the guys. But, I also checked your project, seems like our tools match, and I don't think it would be hard to integrate/do a collaboration. Also you guys seem more experienced than us, so we are open to suggestions  if something comes to your mind in the following days as well.
